libs/sdk-flutter/.gitignore

```diff
@@ -1,2 +1,3 @@
 build/
-.flutter-version
\ No newline at end of file
+.flutter-version
+android/src/main/kotlin/breez_sdk/
```

Comment:
Should we delete and ignore swift bindings as well?

Reply:
Yeah makes sense, will do

Reply:
Good one, ignoring the swift bindings now too.
They weren't committed (I think, or I'm losing my eyesight ;-))